docs/reviewing-a-pull-request.md

```diff
@@ -19,17 +19,30 @@ When reviewing a pull request, always remember the past. That is, remember that
 
 That said, we should not be afraid to change things for the better.
 
+If the tests do change significantly enough to break existing solutions, it's worth calling it out so that we can discuss whether or not the change is worth it in this particular case.
+
 ### Consider the Structural Requirements
 
-We have a tool, [`configlet`](https://github.com/exercism/configlet#configlet), which does track-level linting. This runs on Travis CI when a pull request is submitted. You can also [download it](https://github.com/exercism/configlet/releases/latest) and run it locally.
+We have a tool, [`configlet`](https://github.com/exercism/configlet#configlet), which does track-level linting. This runs on Travis CI when a pull request is submitted.
+
+The tool checks the following (so you don't have to):
+
+- Is the `config.json` file syntactically correct?
+- Does the `config.json` list the correct exercises?
+    - Has every exercise listed been implemented?
+    - Has every implemented exercise been added to the config?
+    - Have any exercises been implemented that shouldn't have been? E.g. exercises that are inappropriate or uninteresting in the language.
+- Is the example solution named with `/example/i` in the path? If not that file will be served when people fetch the exercise.
+
+You can also [download it](https://github.com/exercism/configlet/releases/latest) and run it locally.
+
+Things that the tool does not yet do, and which should be checked:
+
+- Is the test file named correctly, per the guidelines in the language track? (TODO: add to `configlet`)
+
+If there is no automated checking, then you should also verify:
 
-- Is the test file named correctly, per the guidelines in the language track?
-- Is the example solution named with `/example/i` in the path? If not it will be served when people fetch the exercise.
-- Does the directory name of the exercise match that of [an existing, known exercism exercise slug](https://github.com/exercism/x-common/tree/master/exercises)?
```

Comment:
I think I might like to request that this point stay, because I have in recent memory reviewed a PR where the exercise slug was misnamed. I have demonstrated at https://github.com/exercism/xrust/pull/207 (configlet happily accepts).